extensions/wikia/AbTesting/AbTesting.class.php

```diff
@@ -73,7 +60,7 @@ static public function onWikiaSkinTopScripts( &$vars, &$scripts, $skin ) {
 	static public function onWikiaSkinTopShortTTLModules( Array &$modules, $skin) {
 		$app = F::app();
 
-		if ( $app->checkSkin( ['oasis', 'wikiamobile', 'venus'], $skin ) ) {
+		if ( $app->checkSkin( ['oasis', 'wikiamobile'], $skin ) ) {
```

Comment:
It's not that important, but… spaces → `[ 'oasis', 'wikiamobile' ]`. :oncoming_police_car: https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax